common/build/eslint-config-fluid/package.json

```diff
@@ -19,11 +19,12 @@
 		"format": "npm run prettier:fix",
 		"prettier": "prettier --check . --cache --ignore-path ../../../.prettierignore",
 		"prettier:fix": "prettier --write . --cache --ignore-path ../../../.prettierignore",
-		"print-configs": "tsx scripts/print-configs.ts printed-configs",
+		"print-configs": "cross-env ESLINT_USE_FLAT_CONFIG=false tsx scripts/print-configs.ts printed-configs",
```

Comment:
This means that right now, the printed configs are the eslint 8 format. In the next change, we'll switch this to true, and convert the packages to use eslint 9 and the "new" config we've created here. Setting it to true will make the printed configs match what the packages are loading/seeing.